src/main/kotlin/net/developerden/devdenbot/listener/BumpNotificationListener.kt

```diff
@@ -64,6 +64,7 @@ class BumpNotificationListener : EventListener {
         event.channel.sendMessage(
             "${bumpNotificationRole.asMention}, the server is ready to be bumped! **!d bump**"
         ).await()
+        return
```

Comment:
As mentioned in the other PR, this doesn't actually do anything and won't fix the issue